float64 X
float64 Y
float64 Z
float64 RX
float64 RY
float64 RZ
float64 FX
float64 FY
float64 FZ 
float64 theta
